### ISSUE TITLE
[SCILAB] Compilation failure on windows when using the MSVC compiler

### ISSUE THREAD

**Issue by ibell** [swig/swig#333]
On windows, swig 3.0.5, scilab 5.5.1, I get compilation errors with the MSVC compiler.  For what its worth, mingw compiler doesn't work either. See #332 

```
3>c:\users\ihb\desktop\XXXfiles\build\scilab\XXXscilab_wrap.cxx(995): error C4716: 'SWIG_Scilab_Raise_Ex' : must return a value
3>c:\users\ihb\desktop\XXXfiles\build\scilab\XXXscilab_wrap.cxx(1000): error C4716: 'SWIG_Scilab_Raise' : must return a value
```

**Comment by wsfulton** [comment]
This is fixed with commit 0236435c4893185444b6353cf1e3db18153cfb48

**Comment by ibell** [comment]
Thanks!
